SpatialGDK/Documentation/content/get-started/gdk-template.md

```diff
@@ -0,0 +1,152 @@
+# Next steps: Set up the SpatialOS GDK Starter template
+
+Before setting up the SpatialOS GDK Starter template, you need to have followed:
+
+* [Getting started: 1 - Dependencies]({{urlRoot}}/content/get-started/dependencies)
+* [Getting started: 2 - Get and build the SpatialOS Unreal Engine Fork]({{urlRoot}}/content/get-started/tutorial).
+
+If you haven't already, follow our [Multiserver Shooter tutorial]({{urlRoot}}/content/get-started/dependencies) to try out the SpatialOS for Unreal GDK development experience by adding a new feature to a project and testing it across two servers.
+
+If you are ready to start developing your own game with the GDK, follow the steps below. 
+
+### Terms used on this page
+
+* `<Game>` - The directory that contains your project’s .uproject file and Source folder.
+* `<ProjectRoot>` - The directory that contains your `<Game>` directory.
+* `<YourProject>` - The name of your project and .uproject file (for example, `\<Game>\YourProject.uproject`).
+
+### Create a new project using the starter template
+
+After [building the Unreal Engine fork]({{urlRoot}}/content/get-started/build-unreal-fork), in **File Explorer**, navigate to `UnrealEngine\Engine\Binaries\Win64`and double-click UE4Editor.exe to open the Unreal Editor. 
+
+In the [Project Browser](https://docs.unrealengine.com/en-us/Engine/Basics/Projects/Browser) window, click the New Project tab and select the C++ tab, then select the **SpatialOS GDK Starter** template. Choose a suitable directory and name for your project, then click **Create Project**. 
+
+>Note: When you create a project, the Unreal Engine automatically creates a directory named after the project name you entered. This page uses `YourProject` as an example project name.
+
+![The Unreal Engine Project Browser]({{assetRoot}}assets/set-up-template/template-project-browser.png)
+
+*Image: The Unreal Engine Project Browser, with the project file path and project name highlighted.*
+
+After clicking **Create Project**, the Unreal Engine generates the necessary project files before closing the editor and automatically opening Visual Studio. 
```

Comment:
```suggestion
After you have selected **Create Project**, the Unreal Engine generates the necessary project files and directory and then closes the Editor and automatically opening Visual Studio. 
```
Suggest putting it into active voice, simple, and made it clear who does the selecting.

Reply:
```suggestion
After you have selected **Create Project**, the Unreal Engine generates the necessary project files and directories, it then closes the Editor and automatically opens Visual Studio.```

This is a bit of a weird one because _techincally_ the Editor isn't closing itself. Jo's script is closing the editor. But I'm not sure we need to write `The magic script did it` in the docs, as it isn't really relevant.

Reply:
In the situation where we have something automatically happening, I tend to phase in the passive voice (yes, I know...) but add the word "automatically".

In this case, we could do some word yoga...

After you have selected **Create Project**, the Unreal Engine generates the necessary project files and directories. The Editor then automatically closes and opens Visual Studio.